yin_yang/plugins/konsole.py

```diff
@@ -74,6 +74,7 @@ def set_mode(self, dark: bool) -> bool:
             logger.debug(f'Changing profile in konsole session {proc_id}')
             set_profile(f'org.kde.konsole-{proc_id}', profile)
 
+        set_profile('org.kde.konsole', profile) # konsole may don't have session dbus like above
```

Comment:
If possible, it might be a better idea to check if the method above was successful and then execute this alternative.

Reply:
OK, I finally figure it out. When user checked option in konsole.s setting -> Common -> Running all konsole windows in one process, konole's dbus service will not seperate. So yes I should try both way.

Reply:
When that option is checked, I can only toggle current konsole window to change profile but if I open new window or tab, it still remain using old profile. Only if I close all konsole window and then open new one, it will use the new default profile

Reply:
Ok, so we need to check the config file and decide from there